vhdbuilder/scripts/linux/ubuntu/tool_installs_ubuntu.sh

```diff
@@ -140,6 +140,12 @@ installFIPS() {
     echo "Installing FIPS..."
     wait_for_apt_locks
 
+    # installing fips kernel doesn't remove non-fips kernel now, purge current linux-image-azure
+    echo "purging linux-image-azure..."
+    apt_get_purge 5 10 120 linux-image-azure
+    retrycmd_if_failure 120 5 25 apt-mark hold linux-image-azure
+    retrycmd_if_failure 120 5 25 apt-mark hold linux-image-$(uname -r)
```

Comment:
if `apt upgrade` pulls in a newer kernel, this will only purge the currently running kernel, but the other apt package will still exist

you mentioned this scenario should be caught by run test anyway, but jfyi

Reply:
can otherwise grep/wildcard somehow and remove all packages

Reply:
use grep/wildcard to purge & hold all non-fips kernel packages.